app/scripts/templates/sign_up.mustache

```diff
@@ -19,9 +19,13 @@
     {{^error}}
     <div class="error"></div>
     <div class="success"></div>
-    {{#isMigration}}
+    {{#isSyncMigration}}
       <div class="info nudge">{{#t}}Migrate your sync data by creating a new Firefox&nbsp;Account.{{/t}}</div>
-    {{/isMigration}}
+    {{/isSyncMigration}}
+
+    {{#isAmoMigration}}
+        <div class="info nudge">{{#t}}Have an account with a different email? <a href="%(signinUri)s">Sign in</a>{{/t}}</div>
```

Comment:
Need to check if this is a transition to a sign in view or a hard reload into sign in losing all query params